pallets/committee-management/src/impls.rs

```diff
@@ -382,7 +382,7 @@ impl<T: Config> Pallet<T> {
     }
 
     pub(crate) fn calculate_underperforming_validators() {
-        let thresholds = BanConfig::<T>::get();
```

Comment:
Naming: usually we name getter in snake case exactly like its storage is, which would be in that case just `ban_config`. Or do we want to rename storage to `ProducersBanConfig`?

Reply:
I didn't want to rename storage so that we don't have to write a migration. On the other hand, I wanted to differentiate this config from the finality config, hence the name. So I'd rather remove the getter than rename the storage if you don't like violating the convention. WDYT?

Reply:
We need to write migration anyway, so we can rename storage.

Reply:
done